google-http-client/src/test/java/com/google/api/client/http/HttpResponseTest.java

```diff
@@ -461,6 +462,21 @@ public LowLevelHttpResponse execute() throws IOException {
   }
 
   public void testGetContent_gzipEncoding_finishReading() throws IOException {
+    do_testGetContent_gzipEncoding_finishReading("gzip");
+  }
+
+  public void testGetContent_gzipEncoding_finishReadingWithUppercaseContentEncoding() throws IOException {
+    do_testGetContent_gzipEncoding_finishReading("GZIP");
+  }
+
+  public void testGetContent_gzipEncoding_finishReadingWithDifferentDefaultLocaleAndUppercaseContentEncoding() throws IOException {
+    Locale originalDefaultLocale = Locale.getDefault();
+    Locale.setDefault(Locale.JAPAN);
+    do_testGetContent_gzipEncoding_finishReading("GZIP");
+    Locale.setDefault(originalDefaultLocale);
```

Comment:
this probably should be in a finally block so it's still reset if the test fails